### PR TITLE
Added nginx stanza on the compose file

### PR DESCRIPTION
**Pull request by MusaZwane**
I have added the nginx stanza on the compose file at the end.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Maybe change the port mapping to expose in the range we have for the other services

File: docker-compose.yml [hunk 1]
```diff
@@ -57,6 +57,15 @@ services:
...
       - cadvisor
       - node-exporter
+  
+  nginx:
+    image: nginx:latest
+    volumes:
+    - <PATH>/nginx.conf:/etc/nginx/nginx.conf:rw
+    restart: unless-stopped
+    ports:
+    - 5000:80
```
Comment (MusaZwane) [inline docker-compose.yml]: What port do you suggest?

Comment (ghost) [reply docker-compose.yml]: @MusaZwane - see below since our last port was 8082
```suggestion
    - 8083:80
```

**Review by ghost — APPROVED** [review]
All good

**Pull request merged**